Cargo.toml

```diff
@@ -21,13 +21,18 @@ maintenance = { status = "actively-developed" }
 is-it-maintained-issue-resolution = { repository = "virtee/sev" }
 is-it-maintained-open-issues = { repository = "virtee/sev" }
 
+[lib]
+crate-type = ["staticlib", "dylib", "rlib"]
```

Comment:
What's the idea of `dylib` here? Unfortunately it breaks builds with `panic="abort"`.

Reply:
Good point, `dylib` is really for macOS which isn't really a concern here. I think it's safe to remove this. @larrydewey do you agree?